src/components/HomeSearchBox/index.tsx

```diff
@@ -6,9 +6,29 @@ const HomeSearchBox = () => {
   const { t } = useTranslation('home')
   const router = useRouter()
   const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filtered, setFiltered] = useState<string[]>([])
+  const [isShow, setIsShow] = useState(false)
 
   const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchValue(event.target.value)
+    const suggestions : string[] = ['Alabama','Alaska','American Samoa', 'Bubble Holmes', 'Contra Hits', 'Amazon'];
+    const searchValue = event.currentTarget.value
```

Comment:
The variable name is declared above, this could cause a bug. Better rename it to something like `searchInput` or any other meaningfuls.